index.js

```diff
@@ -52,6 +52,23 @@ function toggleSound(){
         timer.innerHTML = "<b>" + counter + "</b>";
     }, 1000);
 
+    // refresh when reload
+    const showAllCards = ()=>{
+        cards.forEach((card, i)=>{
+            card.children[0].src=cardImageSrcs[i]; // Setting image source for flipped front face

```

Comment:
`cardImageSrc` contain all the images , but the actual images are shown using `cardArray`
this Is causing a bug that the image you are showing are not in order what actually in the game

Reply:
this issue I already resolved, in updated code
https://github.com/ContriHUB/memory-game/pull/26/commits/e77281f72f25c03c8ce50ab3403775ce7b58a8da